### ISSUE TITLE
color_by argument

### ISSUE THREAD

**Issue by allmanbrent** [boaz85/pymuller#3]
hello, could you please add an example of what the color_by object should look like? I attempted this `color_by = pd.Series([.1,.2,.3,.4,.5], index = [' GGTA', ' GGTT',' GACA', ' AACA', ' AATA'])`. So what are the values of color_by supposed to be? I keep getting errors. Most recent one is `TypeError: ufunc 'isfinite' not supported for the input types, and the inputs could not be safely coerced to any supported types according to the casting rule ''safe''`

My adjacency matrix looks like 
```
adjacency_mat
  Parent Identity
1   GGTA     GGTT
2   GGTA     GACA
3   GGTA     AACA
4   GGTA     AATA
```

and my population matrix looks like 
```
Generation Identity  Population
0          28     GGTA    0.969828
1          39     GGTA    0.000000
2          53     GGTA    0.000000
3          75     GGTA    0.068991
4          28     GGTT    0.001078
5          39     GGTT    0.000000
6          53     GGTT    0.000000
```

**Comment by LukasHafner** [comment]
Issue is persisting. Have you found a solution @allmanbrent ?

**Comment by allmanbrent** [comment]
No, I have not. Sadly I had to move on to a different visualization solution because I couldn't figure this out. Instead, I just went with a scatter plot, some text description, and an accompanying cartoon.

**Comment by Entropy5** [comment]
added an example
good luck
https://github.com/boaz85/pymuller/pull/4
